src/me/StevenLawson/TotalFreedomMod/Commands/Command_opall.java

```diff
@@ -30,11 +28,21 @@ else if (args[0].equals("-s"))
                 targetGamemode = GameMode.SURVIVAL;
             }
         }
-
+        
```

Comment:
Not sure why this is here.. Its not a big deal, but just no idea why it would be here.